src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -16,6 +16,15 @@ import { getFocusedElementProp } from "../../tests/utils";
 import { skipAnimations } from "../../tests/utils";
 import { html } from "../../../support/formatting";
 
+async function getInputValue(page: E2EPage): Promise<string> {
+  return await page.evaluate(() => {
```

Comment:
No need to await here since you're returning the promise.

Reply:
nice catch there, thank you!